src/pd/util.rs

```diff
@@ -393,7 +393,7 @@ pub fn validate_endpoints(
     match members {
         Some(members) => {
             let (client, members) = try_connect_leader(Arc::clone(&env), security_mgr, &members)?;
-            info!("All PD endpoints are consistent: {:?}", cfg.endpoints);
+            info!("all PD endpoints are consistent"; "endpoint" => ?cfg.endpoints);
```

Comment:
```suggestion
            info!("all PD endpoints are consistent"; "endpoints" => ?cfg.endpoints);
```
Since `cfg.endpoints` is a `Vec`, I think `+s` is better.